datasets/elkarhizketak/README.md

```diff
@@ -0,0 +1,170 @@
+
+Then, dialogues were collected during some online sessions that we arranged with Basque speaking volunteers. The dialogues involve two crowd workers: (1) a student ask questions after reading a small introduction about the person, but without seeing the section text; and (2) a teacher answers the questions selecting a span of text of the section.
+
+#### Who are the source language producers?
+
+The language producers are Basque speaking volunteers which hold a conversation using a text-based chat interface developed for those purposes.
+
+### Annotations
+
+#### Annotation process
+
+[More Information Needed]
+
+#### Who are the annotators?
+
+[More Information Needed]
+
+### Personal and Sensitive Information
+
+[More Information Needed]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[More Information Needed]
+
+### Discussion of Biases
+
+[More Information Needed]
+
+### Other Known Limitations
+
+[More Information Needed]
+
+## Additional Information
+
+### Dataset Curators
+
+The dataset was created by Arantxa Otegi, Jon Ander Campos, Aitor Soroa and Eneko Agirre from the [HiTZ Basque Center for Language Technologies](https://www.hitz.eus/) and [Ixa NLP Group](https://www.ixa.eus/) at the University of the Basque Country (UPV/EHU).
+
+### Licensing Information
+
+Copyright (C) by Ixa Taldea, University of the Basque Country UPV/EHU.
+
+This dataset is licensed under the Creative Commons Attribution-ShareAlike 4.0 International Public License (CC BY-SA 4.0).
+To view a copy of this license, visit [https://creativecommons.org/licenses/by-sa/4.0/legalcode](https://creativecommons.org/licenses/by-sa/4.0/legalcode).
+
+### Citation Information
+
+If you are using this dataset in your work, please cite this publication:
+
+@inproceedings{otegi-etal-2020-conversational,
+    title = "{Conversational Question Answering in Low Resource Scenarios: A Dataset and Case Study for Basque}",
+    author = "Otegi, Arantxa  and
+      Agirre, Aitor  and
+      Campos, Jon Ander  and
+      Soroa, Aitor  and
+      Agirre, Eneko",
+    booktitle = "Proceedings of the 12th Language Resources and Evaluation Conference",
+    year = "2020",
+    address = "Marseille, France",
+    publisher = "European Language Resources Association",
+    url = "https://aclanthology.org/2020.lrec-1.55",
+    pages = "436--442"
+}
```

Comment:
````suggestion
```
@inproceedings{otegi-etal-2020-conversational,
    title = "{Conversational Question Answering in Low Resource Scenarios: A Dataset and Case Study for Basque}",
    author = "Otegi, Arantxa  and
      Agirre, Aitor  and
      Campos, Jon Ander  and
      Soroa, Aitor  and
      Agirre, Eneko",
    booktitle = "Proceedings of the 12th Language Resources and Evaluation Conference",
    year = "2020",
    address = "Marseille, France",
    publisher = "European Language Resources Association",
    url = "https://aclanthology.org/2020.lrec-1.55",
    pages = "436--442"
}
```
````